### PR TITLE
[FLANE-134] add translates to Spanish(es_419) and Portugales(pt_BR)

### PR DESCRIPTION
**Pull request by yozhsh**
**Description:** add translates from [file](https://github.com/raccoongang/edx-platform/files/2880175/Traduccion.7.xlsx)
  to es_419 and pt_BR(djangojs.po, django.po) 

**Youtrack:** https://youtrack.raccoongang.com/issue/FLANE-134

### CODE REVIEW

File: conf/locale/pt_BR/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -10789,3 +10789,8 @@ msgstr ""
 #: cms/templates/js/video/transcripts/messages/transcripts-use-existing.underscore
 msgid "Use Current Transcript"
 msgstr "Usar a Transcrição Atual"
+
+
+#: lms/static/templates/instructor/instructor_dashboard_2/cohorts.underscore
+msgid "To review student cohort assignments or see the results of uploading a CSV file, download course profile information or cohort results on the {link_start}Data Download{link_end} page."
+msgstr "para revisar as atribuições de grupo dos alunos ou ver os resultados de fazer upload de um arquivo SCV, fazer o download das informações do perfil do curso ou grupo os resultados na página de {link_start}download de dados{link_end}"
```
Comment (sendr) [inline conf/locale/pt_BR/LC_MESSAGES/djangojs.po]: add empty string please

File: conf/locale/es_419/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -19018,10 +19016,8 @@ msgid ""
 "When you are ready to generate certificates for your course, click Generate "
 "Certificates. You do not need to do this if you have set the certificate "
 "mode to on-demand generation."
-msgstr ""
-"Cuando esté listo para generar los certificados de su curso, haga clic en "
-"Generar Certificados. No tendrá que hacer esto si ha seleccionado el modo de"
-" certificados por demanda."
+msgstr "When you are ready to generate certificates for your course, click Generate Certificates. You do not need to do this if you ave set the certificate mode to on-demand generation."
```
Comment (sendr) [inline conf/locale/es_419/LC_MESSAGES/django.po]: make all long lines according to PEP requirements
line above is very good example how to do this

File: conf/locale/pt_BR/LC_MESSAGES/djangojs.po (hunk 1 above)
Comment (sendr) [reply conf/locale/pt_BR/LC_MESSAGES/djangojs.po]: @yozhsh 
http://prntscr.com/mnrv0b

File: conf/locale/es_419/LC_MESSAGES/django.po (hunk 2 above)
Comment (yozhsh) [reply conf/locale/es_419/LC_MESSAGES/django.po]: @sendr 
Done

File: conf/locale/pt_BR/LC_MESSAGES/djangojs.po (hunk 1 above)
Comment (yozhsh) [reply conf/locale/pt_BR/LC_MESSAGES/djangojs.po]: @sendr 
Done

File: conf/locale/es_419/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -18991,19 +18991,17 @@ msgstr "Deshabilitar la generación de certificados por estudiantes"
...
+msgstr "Habilitar certificados generados por estudiantes"
 
 #: lms/templates/instructor/instructor_dashboard_2/certificates.html
 msgid ""
 "You must successfully generate example certificates before you enable "
 "student-generated certificates."
-msgstr ""
-"Debe generar exitosamente certificados de ejemplo antes de habilitar la "
-"opción para que los estudiantes generen certificados."
+msgstr "Debe generar certificados de ejemplo con éxito antes de habilitar los certificados generados por los estudiantes."
```
Comment (a-kryachko) [inline conf/locale/es_419/LC_MESSAGES/django.po]: to split long lines in `msgstr` to be like `msgid`

File: conf/locale/es_419/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -19018,10 +19016,9 @@ msgid ""
 "When you are ready to generate certificates for your course, click Generate "
 "Certificates. You do not need to do this if you have set the certificate "
 "mode to on-demand generation."
-msgstr ""
-"Cuando esté listo para generar los certificados de su curso, haga clic en "
-"Generar Certificados. No tendrá que hacer esto si ha seleccionado el modo de"
-" certificados por demanda."
+msgstr "When you are ready to generate certificates for your course, click Generate Certificates. You do not need "
+"to do this if you ave set the certificate mode to on-demand generation."
+
```
Comment (a-kryachko) [inline conf/locale/es_419/LC_MESSAGES/django.po]: to remove extra emty lines

File: conf/locale/es_419/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -19226,21 +19221,15 @@ msgid ""
...
 "background, meaning it is OK to navigate away from this page while your "
 "report is generating."
-msgstr ""
-"Para los cursos más grandes, la generación de algunos reportes puede tomar "
-"varias horas. Cuando un reporte haya sido generado, un vínculo con la fecha "
-"y la hora de generación aparecerá en la tabla a continuación. Estos reportes"
-" se generan en segundo plano, así que puede salir de esta página o visitar "
-"otras paginas mientras el reporte es generado."
+msgstr "Para cursos grandes, generar algunos informes puede tomar varias horas. Cuando se completa la generación del informe, en la siguiente tabla aparece un enlace que incluye la fecha y la hora de la generación. Estos informes se generan en segundo plano, lo que significa que puedes navegar mientras se genera tu informe"
+
```
Comment (a-kryachko) [inline conf/locale/es_419/LC_MESSAGES/django.po]: to remove empty line.
to split long string

File: conf/locale/es_419/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -19226,21 +19221,15 @@ msgid ""
...
 #: lms/templates/instructor/instructor_dashboard_2/data_download.html
 msgid ""
 "Please be patient and do not click these buttons multiple times. Clicking "
 "these buttons multiple times will significantly slow the generation process."
-msgstr ""
-"Por favor sea paciente y no haga clic en el botón varias veces, ya que esto "
-"reducirá significativamente la velocidad del proceso de generación del "
-"reporte."
+msgstr "Por favor sea paciente y no haga clic en estos botones varias veces. Al hacer clic en estos botones varias veces se ralentizará significativamente el proceso de generación."
+
```
Comment (a-kryachko) [inline conf/locale/es_419/LC_MESSAGES/django.po]: same here

**Pull request merged**